lm75bd/lm75bd.c

```diff
@@ -25,10 +25,24 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
   return ERR_CODE_SUCCESS;
 }
-
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  //sets the temp register
+  error_code_t errCode;
+
+  if ( temp == NULL){
+    return ERR_CODE_INVALID_ARG;
+  }
+  uint8_t buff= LM75BD_REG_TEMP_REGISTER; 
+  errCode = i2cSendTo(devAddr, &buff, 1); // this sets the temp register, there is only one byte being passed in to set the pointer register
+  if (errCode != ERR_CODE_SUCCESS) return errCode;
```

Comment:
You could've used RETURN_IF_ERROR_CODE which would log and return the error code
```suggestion
  RETURN_IF_ERROR_CODE(i2cSendTo(devAddr, &buff, 1));
```